src/modules/form/components/DynamicField.tsx

```diff
@@ -304,6 +309,21 @@ const DynamicField: React.FC<DynamicFieldProps> = ({
           />
         </InputContainer>
       );
+    case ItemType.TimeOfDay:
```

Comment:
This needs to be added to storybook but I can do it as part of https://github.com/greenriver/hmis-frontend/pull/657 after this merges, so we dont need to deal with merge conflicts

Reply:
Huh, I'd kind of forgotten storybook was even here tbh. Storybook does seem to work for me locally on this branch, not sure what the difference is.

Thanks for the update/fix in https://github.com/greenriver/hmis-frontend/pull/659 but I do want to call out that the MUI upgrade from 5.11 to 5.15 includes breaking changes (check the MUI [github releases pages going back to 5.12](https://github.com/mui/material-ui/releases?page=1)). I haven't reviewed those release notes yet but wanted to make sure that was okay

Reply:
Hmm okay. If its working for you, lets not merge my PR. I'll keep plugging away at the build error I was getting on this branch

Reply:
Huh ok good to know. I'll review the release notes to see

Reply:
After doing a full yarn cache cleanout and reinstalling, its working for me now. I will close that PR, and we can update those packages separately. thanks!